app/store/sagas/appSagas/login.js

```diff
@@ -49,12 +50,15 @@ export function* authenticationFlowSaga() {
         // eslint-disable-next-line no-unused-vars, prefer-const
         let { token, error } = yield call(authorize, navigation.params.code);
         if (!error) {
+          const task = yield spawn(watchDatabase);
```

Comment:
this shouldn t be here. this has nothing to do with login with strava 
watch for a successfull login et logout to start watchdatabase